instrument.yml

```diff
@@ -0,0 +1,11 @@
+name: Honeydew
+run: onEach
+results:
+  - dir: ${instrument}/results
+    files:
```

Comment:
Remove the files attribute.

Reply:
Honeydew generates the json model and some csv files with structural relations.  We want to keep both :)